test/integration/targets/ec2_group/tasks/main.yml

```diff
@@ -11,6 +11,7 @@
 - include: ./credential_tests.yml
 - module_defaults:
     group/aws:
+      debug_botocore_endpoint_logs: True
```

Comment:
Once #51149 is merged I'll be able to remove this. Right now any tests setting module_defaults for group/aws (such as this one) overwrite the setting from the test runner.